sema/subyt/sources.py

```
@@ -286,7 +337,7 @@
                     data = [data]
             return iter(data)
 
-        def __exit__(self):
+        def __exit__(self, *exc):
```

Ruff (0.8.2)

Missing return type annotation for special method `__exit__`

(ANN204)